opensrp-reveal/src/test/java/org/smartregister/reveal/activity/OfflineMapsActivityTest.java

```diff
@@ -0,0 +1,99 @@
+package org.smartregister.reveal.activity;
+
+import com.mapbox.mapboxsdk.offline.OfflineManager;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
+import org.smartregister.family.adapter.ViewPagerAdapter;
+import org.smartregister.reveal.BaseUnitTest;
+import org.smartregister.reveal.fragment.AvailableOfflineMapsFragment;
+import org.smartregister.reveal.fragment.DownloadedOfflineMapsFragment;
+import org.smartregister.reveal.model.OfflineMapModel;
+import org.smartregister.reveal.util.TestingUtils;
+import org.smartregister.reveal.view.OfflineMapsActivity;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by Richard Kareko on 1/30/20.
+ */
+
+public class OfflineMapsActivityTest extends BaseUnitTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+
+    @Mock
+    private ViewPagerAdapter adapter;
+
+    @Mock
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)